### ISSUE TITLE
[pacman-boot] boot times

### ISSUE THREAD

**Issue by udeved** [manjaro/manjaro-tools#112]
I think the boot takes way too long with a full mirrorlist.
Should we probably map the language selection to a mirror country?
Ie we would boot livecd with a "localized" mirrorlist.
Do we have enough mirrors to map this?

**Comment by philmmanjaro** [comment]
I've to check if pacman-boot get's called after all ...

**Comment by udeved** [comment]
It gets called.
I just saw you did snapshot.
Now the fix for pacman-boot script is in all branches.
To explain, 0.8.13 had a bug, and pacman-mirrors and sync was never run.
https://github.com/manjaro/manjaro-tools-livecd/commit/146f7cb73973c9132eea35824e11793744e83b27

**Comment by Kirek** [comment]
pacman-boot is broken in systemd (it doesn't get called, no service named pacman-boot), and we are lucky it is because this little script can easily add a few minutes to the boot times (at least 1-2 minute checking the mirrors, an then a few more minutes downloading the databases, because sometimes lowest ping doesn't mean best speed).

tl;dr: pacman-boot should be completely removed.

After all when starting the desktop pamac or octopi will refresh the database again.

That being said building a localized mirrorlist depending on the selected language is not a bad idea, but should be done **without pinging anything**, and if no keyboard is selected (efi boot), we should randomize the mirrorlist so it doesn't always default to the africa mirror, which is pretty slow for the majority of people.
This way when the desktop boots, and pamac/octopi refresh the database it won't be as slow as it is right now.

**Comment by udeved** [comment]
I guess the net profiles need the pacman-boot as the only editions.

So the pacman-boot service is not loaded or run on systemd at all?
The boot time about 2 minutes is what it takes with a working pacman-boot for me.
This must be reduced drastically. The iso boot nice and fast, but the pacman-boot is ugly with full mirrorlist.

**Comment by Kirek** [comment]
Simply removing pacman-boot its the best option. The fact that it wasn't working in all in the systemd editions and no one complained/noticed makes clear that it was useless to begin with.
We don't want people to think that Manjaro is slow because it takes ages to boot the livecd because we are doing something that is basically unnecessary.

I'm removing it.

Then if we can generate a better mirrorlist depending on language/keyboard selection will be better (but still not essential...).

**Comment by philmmanjaro** [comment]
I removed the service, I think couple of stable builds ago. Around 0.8.10 or so. Don't remember it. Removing it is the way to go here ...

**Comment by Kirek** [comment]
I suppose this can be closed then.
